.github/dependabot.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+
+registries:
+  npm-github:
+    type: npm-registry
+    url: https://npm.pkg.github.com
+    token: ${{ secrets.NPM_PKG_GITHUB_TOKEN }}
+
+updates:
+  - package-ecosystem: "npm"
+    registries:
+      - npm-github
+    directory: "/"
+    groups:
+      toolchain:
+        update-types:
+          - "minor"
+          - "patch"
+        patterns:
+          - "*vite*"
+          - "depcheck"
+          - "dprint"
+          - "eslint*"
+          - "ts-node"
+          - "typescript"
+      fluent:
+        update-types:
+          - "minor"
+          - "patch"
+        patterns:
+          - "*fluentui*"
+      dev-dependencies:
+        dependency-type: "development"
+        update-types:
+          - "minor"
+          - "patch"
+        # FIXME: can be removed after https://github.com/dependabot/dependabot-core/issues/7915
```

Comment:
That issue seems to be fixed... ?

Reply:
Yes, but I don't know when it's in production.

Reply:
Ahh, forgot to check if it was release. Sorry. :smiley: